### DISCUSSION TITLE
Run command and [one time initial set up] commands would be helpful for people trying to get started:

### DISCUSSION THREAD

**Discussion by scaseye** [filecoin-project/lotus#7770] [Storage Provider] [answered]
@stuberman was this documentation request done and if so where can I find it?

you posted this Aug 2020

Run command and [one time initial set up] commands would be helpful for people trying to get started:

[Build commands]
Describe or link to instructions

**Lotus Daemon**
1. Start Lotus daemon - `nohup lotus daemon > ~/lotus.log 2>&1 &`
Check logs - `tail -f lotus.log` or `tail -1000 lotus.log`
If you have not preconfigured your Lotus daemon port: `lotus net listen`
2. Sync Lotus to blockchain `lotus sync wait`
[`lotus wallet new bls `]
[Fund wallet if new and check balance]

**Lotus Miner**
On first start only [`lotus-miner init --no-local-storage --sector-size=32GiB --owner=t3...`]
1. Start Miner `nohup lotus-miner run > ~/miner.log 2>&1 &`
Check logs - `tail -f miner.log` or `tail -1000 miner.log`
*** Wait 5 minutes for miner to start ***
If you have not preconfigured your Lotus miner port: `lotus-miner net listen`
[Put public address on blockchain`lotus-miner actor set-addrs /ip4/<public IP>/tcp/<public port>`]
[Set deal prices `lotus-miner storage-deals set-ask --price n --verified-price n --max-piece-size 32GiB`]
[Specify storage path `lotus-miner storage attach --init --store /mypath/storage`]

**Lotus Worker(s)**
1. Start worker(s) `lotus-worker --worker-repo /Lotus/worker run --listen 127.0.0.1:3456 > ~/worker.log 2>&1 &`
(Provide details to add additional local or remote sealing workers...)

**On going operational guide**
How do you stop or pause a process?
Are there dependencies when trying to stop a process?
What, if anything, needs to be done before you reboot a system?
How do you temporarily stop accepting deals?
What is the impact if you are offline for a little while?
Process to add new hardware components (drives, GPUs, remote workers, etc)

_Originally posted by @stuberman in https://github.com/filecoin-project/lotus/issues/2956#issuecomment-673161134_

**Comment by stuberman** [comment]
That brings back memories... 
Some of this is incorporated into the latest [Lotus docs](https://docs.filecoin.io/mine/)
We are also creating tutorials and guides to help fellow miners [here](https://github.com/filecoin-project/lotus/discussions) and [here](https://github.com/filecoin-project/community/discussions).

**Comment by stuberman** [comment, marked as answer]
Some quick updates though:

1. There is currently no stop command for lotus-worker - use `kill [pid]` after finding the worker pid
2. Since command syntax changes, use `lotus --help` to walk into the sub commands like `lotus sync --help`
3. One way to stop accepting deals: `lotus-miner storage-deals selection reject --help` or simply raise prices

@TippyFlits created this nice reference: https://github.com/filecoin-project/community/discussions/143
